orders/tests.py

```diff
@@ -0,0 +1,62 @@
+import datetime
+from django.utils import timezone
+from dogowner.models import DogOwner
+from daycare.models import DayCare
+from .models import Order
+from .models import StatusOptions
+import pytest
+
+NUMBER_OF_DAYS = 3
+STARTING_DATE = timezone.now()
+FINISHING_DATE = timezone.now() + datetime.timedelta(days=NUMBER_OF_DAYS)
+PRICE_PER_DAY = 100
+STATUS = StatusOptions.Pending
+
+
+@pytest.fixture
+def create_order():
+    return Order.create(dog_owner_id=DogOwner.objects.get(id=1),
+                        daycare_id=DayCare.objects.get(id=1),
+                        starting_date=STARTING_DATE,
+                        finishing_date=FINISHING_DATE,
+                        price_per_day=PRICE_PER_DAY,
+                        )
+
+
+@pytest.mark.django_db
+class TestOrderModel:
+    def test_create_order(self):
+        new_order = Order.create(dog_owner_id=DogOwner.objects.get(id=1),
```

Comment:
why do we create a new order here instead of using the fixture above?

Reply:
You're right, I'll change the test to check if the creation of the fixture succeeded.